doc/source/user_guide/index.rst

```diff
@@ -26,259 +24,60 @@ Python code to control and monitor Ansys Fluent.
    solution
 
 
-Overview
---------
-You use the :func:`launch_fluent() <ansys.fluent.core.launcher.launcher.launch_fluent>` method
-to launch an instance of Fluent that runs as a server in the background.
+PyFluent User Guide
+-------------------
+Welcome to the PyFluent User Guide. This guide will help you understand how to use PyFluent to leverage the power of Ansys Fluent for your CFD simulations.
 
-You can launch Fluent in solution mode with this code:
 
-.. code:: python
+A Simple Example
+----------------
 
-    from ansys.fluent.core import launch_fluent
+.. code-block:: python
 
-    solver = launch_fluent(mode="solver")
+  >>> import ansys.fluent.core as pyfluent
+  >>> meshing = pyfluent.launch_fluent(mode=pyfluent.FluentMode.MESHING, product_version=pyfluent.FluentVersion.v242)
+  >>> watertight = meshing.watertight()
+  >>> watertight.import_geometry.file_name = pyfluent.examples.download_file("mixing_elbow.pmdb","pyfluent/mixing_elbow")
+  >>> watertight.import_geometry()
+  >>> watertight.create_volume_mesh()
+  >>> meshing.switch_to_solver()
+  >>> solver.setup.boundary_conditions.set_zone_type(zone_list=["cold-inlet", "hot-inlet"], new_type="velocity-inlet")
+  >>> solver.setup.boundary_conditions.set_zone_type(zone_list=["outlet"], new_type="pressure-outlet")
+  >>> solver.setup.cell_zone_conditions.set_zone_type(zone_list="elbow-fluid", new_type="fluid")
+  >>> solver.solution.initialization.hybrid_initialize()
+  >>> solver.solution.run_calculation.iterate(iter_count=100)
+  >>> velocity_data = solver.field_data.get_vector_field_data(field_name="velocity", surface_name="cold-inlet")
 
-You can launch Fluent in meshing mode with this code:
 
-.. code:: python
+Key Features
+------------
 
-    from ansys.fluent.core import launch_fluent
+Launching Fluent from PyFluent
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Easily :ref:`start a Fluent session <_ref_user_guide_launch>` locally or remotely with a variety of launch options, or connect
+to an existing session. Interact with Fluent through the session object returned by the launch
+or connect methods.
 
-    meshing = launch_fluent(mode="meshing")
+Guided Meshing Workflows
+~~~~~~~~~~~~~~~~~~~~~~~~
+Leverage intuitive, :ref:`guided workflows <_ref_meshing_workflow>` to create high-quality meshes.
 
+Solution Mode and Settings Objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Utilize :ref:`settings objects <_ref_settings>` to configure and control your simulation.
 
-For more information, see :ref:`ref_user_guide_launch`
-and :ref:`ref_launcher`.
+Data Extraction
+~~~~~~~~~~~~~~~
+Easily extract solution and mesh data for analysis and post-processing.
```

Comment:
I'd prefer to avoid the qualification `Easily`. The usage is easy in its own right, as users will find.